src/popup/pages/ShowMnemonic/index.scss

```diff
@@ -26,7 +26,7 @@
 
 .mne-item {
   padding: 8px 15px;
-  margin: 5px 15px 5px 0;
+  margin: 5px 10px 5px 0;
```

Comment:
move these to the styling commit? or is this something else

Reply:
Split into two commits, and renamed a bit:
```diff
- Generate 24-word mnemonics
+ Generate 24-word mnemonics
+ Display smaller mnemonic words to fit average 24-word mnemonics

 Simplify .mne-item styling
 
- Make popup wider so 24-word mnemonics don't overflow
+ Make popup wider to fit widest 24-word mnemonics
```